dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/README.md

```diff
@@ -382,7 +382,7 @@ Code without diagnostic:
 var x = items.OrderBy(f => f.Surname).ThenByDescending(f => f.Name);
 ```
 
-## ACL1008 - Controller actions have ProducesResponseType attribute
+## ACL1008 - Controller actions have ProducesResponseType attribute when return type is not TypedResults
```

Comment:
I think [HelpLinkUrlFactory](https://github.com/audaciaconsulting/Audacia.CodeAnalysis/blob/master/dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Common/HelpLinkUrlFactory.cs) will need to be updated to reflect this title change